website/files/models/base.py

```diff
@@ -199,7 +199,7 @@ class StoredFileNode(StoredObject, Commentable):
     checkout = fields.AbstractForeignField('User')
 
     #Tags for a file, currently only used for osfStorage
-    tags = fields.ForeignField('Tag', list=True, index=True)
+    tags = fields.ForeignField('Tag', list=True)
```

Comment:
I know, I saw the link in the commit message. But what does it mean as far as performance? Seto can probably answer the question quickly.